src/__tests__/linter.test.ts

```diff
@@ -146,6 +146,43 @@ describe('linter', () => {
     expect(result[0]).toHaveProperty('severity', DiagnosticSeverity.Hint);
   });
 
+  test('should support human readable severity levels', async () => {
+    spectral.addFunctions(oas2Functions());
+
+    spectral.addRules({
```

Comment:
Does it make sense to just test the `off` severity here as well?

Reply:
Totally, the more tests covering rulesets, the better.

Reply:
Ha, so actually programmatic rules insertion does not support that yet since I made this change apply to rulesets only.
```ts
 public addRules(rules: RuleCollection) {
    Object.assign(this._rules, merge({}, rules));
  }

  public mergeRules(rules: PartialRuleCollection) {
    for (const ruleName in merge({}, rules)) {
      if (!rules.hasOwnProperty(ruleName)) continue;
      const rule = rules[ruleName];
      if (rule) {
        this._rules[ruleName] = merge(this._rules[ruleName], rule);
      }
    }
  }
```
Should we support it?

Reply:
Nah I think if somebody is programmatically adding rules they know what they’re doing. We are deprecating these methods soon anyway.

Reply:
Alrighty, leaving this as is.